### ISSUE TITLE
Request to add custom functions to MathNotepad

### ISSUE THREAD

**Issue by plutoniumm** [josdejong/mathjs#2409]
I can understand why the MathNotebook is closed source and I'm not requesting for it to be open sourced, however I would request the authors to make for a provision/feature where we can define custom functions

A white back when I was studying Numerical Analysis, while not being the right language for the given problem I chose to use JS because of my comfort level with it and Notebook was a instrumental in that, since then its been of frequent use to me.

I would like to propose a feature in which end user can upload a JS file (saved locally as say PWA/in storage or something) which can then be used to call for written functions, (like say ive wanted to do FCT and Fast Fourier Recently). Locally saving it will ensure safety will allowing end user to speed up work

**Comment by josdejong** [comment]
yes definitely! Thanks for your suggestion Manav. I didn't have much time to work on MathNotepad last couple of years, but expect to be able to pick up things like this this year.
